official/resnet/cifar10_main.py

```diff
@@ -138,39 +138,37 @@ def train_preprocess_fn(image, label):
 
 
 def input_fn(is_training, data_dir, batch_size, num_epochs=1):
-  """Input_fn using the contrib.data input pipeline for CIFAR-10 dataset.
+  """Input_fn using the tf.data input pipeline for CIFAR-10 dataset.
 
   Args:
     is_training: A boolean denoting whether the input is for training.
+    data_dir: The directory containing the input data.
+    batch_size: The number samples per batch.
```

Comment:
Thanks for fixing this! Just need to fix "number samples" to "number of samples"

Reply:
Good catch, fixed